docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-apache.asciidoc

```diff
@@ -16,66 +16,37 @@ These {anomaly-jobs} find unusual activity in HTTP access logs.
 For more details, see the {dfeed} and job definitions in
 https://github.com/elastic/integrations/blob/{branch}/packages/apache/kibana/ml_module/apache-Logs-ml.json[GitHub].
 Note that these jobs are available in {kib} only if data exists that matches the
-{dfeed} query.
-
-low_request_rate_apache::
-Detects low request rates.
-
-Job details:::
-
-* Analyzes request rates (using the <<ml-count,`low_count` function>>).
-
-Required {beats} or {agent} integrations:::
-
-* Apache integration
-
-source_ip_request_rate_apache::
-Detects unusual source IPs.
-
-Job details:::
-
-* Analyzes request rates (using the <<ml-count,`high_count` function>>)
-relative to all the source IPs (`over_field_name` is `source.address`).
-
-Required {beats} or {agent} integrations:::
-
-* Apache integration
-
-source_ip_url_count_apache::
-Detects unusual source IPs.
-
-Job details:::
-
-* Analyzes distinct counts of URLs (using the
-<<ml-distinct-count,`high_distinct_count` function>> on the `url.original`
-field) relative to all the source IPs (`over_field_name` is `source.address`).
-
-Required {beats} or {agent} integrations:::
-
-* Apache integration
-
-status_code_rate_apache::
-Detects unusual status code rates.
-
-Job details:::
-
-* Analyzes request rates (using the <<ml-count,`count` function>>) split by
-status code (`partition_field_name` is `http.response.status_code`).
-
-Required {beats} or {agent} integrations:::
-
-* Apache integration
-
-visitor_rate_apache::
-Detects unusual visitor rates.
-
-Job details:::
-
-* Analyzes request rates using the <<ml-nonzero-count,`non_zero_count` function>>.
-
-Required {beats} or {agent} integrations:::
-
-* Apache integration
+query specified in the
+https://github.com/elastic/integrations/blob/{branch}/packages/apache/kibana/ml_module/apache-Logs-ml.json#L11[manifest file].
```

Comment:
These URLs might need to be explicitly overridden, since there's no 8.0 branch in the integrations repo:
```suggestion
https://github.com/elastic/integrations/blob/main/packages/apache/kibana/ml_module/apache-Logs-ml.json#L11[manifest file].
```